circuits/ts/index.ts

```diff
@@ -95,4 +103,12 @@ const extractVk = async (zkeyPath: string) => {
     return vk
 }
 
+/**
+ * Check if we are running on an arm chip
+ * @returns whether we are running on an arm chip
+ */
+const isArm = (): boolean => {
```

Comment:
nit: Since WASM is platform-independent, there is no need to specifically check for the arm architecture. It's worth considering that there are other platform architectures, such as RISC-V (riscv64)